tests/unit/backend/data/test_LocalDataService.py

```diff
@@ -239,8 +298,28 @@ def test_readWriteCalibrationRecordV2():
             CalibrationRecord.parse_raw(Resource.read("inputs/calibration/CalibrationRecord.json"))
         )
         actualRecord = localDataService.readCalibrationRecord("57514")
-        shutil.rmtree(Resource.getPath("outputs/v_1"))
-        shutil.rmtree(Resource.getPath("outputs/v_2"))
+        # shutil.rmtree(Resource.getPath("outputs/v_1"))
+        # shutil.rmtree(Resource.getPath("outputs/v_2"))
+
+        assert actualRecord.runNumber == "57514"
+
+    def test_readWriteNormalizationRecord():
+        localDataService = LocalDataService()
+        localDataService.instrumentConfig = mock.Mock()
+        localDataService._generateStateId = mock.Mock()
+        localDataService._generateStateId.return_value = ("123", "456")
+        localDataService._readReductionParameters = mock.Mock()
+        localDataService._constructCalibrationStatePath = mock.Mock()
+        localDataService._constructCalibrationStatePath.return_value = Resource.getPath("outputs/")
+        localDataService.writeNormalizationRecord(
+            NormalizationRecord.parse_raw(Resource.read("inputs/normalization/NormalizationRecord.json"))
+        )
+        localDataService.writeNormalizationRecord(
+            NormalizationRecord.parse_raw(Resource.read("inputs/normalization/NormalizationRecord.json"))
+        )
+        actualRecord = localDataService.readNormalizationRecord("57514")
+        # shutil.rmtree(Resource.getPath("outputs/v_1"))
+        # shutil.rmtree(Resource.getPath("outputs/v_2"))
```

Comment:
As before, these lines are important

Reply:
There lines were causing a lot of other tests to fail. As some other tests require the presence of these directories. Meant to add them to the last test so that these would be deleted after the pertinent tests were complete.

Reply:
I will push a fix using temp directories